gems/ibm_cloud_power/lib/ibm_cloud_power/models/create_image.rb

```diff
@@ -186,8 +186,7 @@ def valid?
       return false if @source.nil?
       source_validator = EnumAttributeValidator.new('String', ["root-project", "url"])
       return false unless source_validator.valid?(@source)
-      os_type_validator = EnumAttributeValidator.new('String', ["aix", "ibmi", "redhat", "sles"])
-      return false unless os_type_validator.valid?(@os_type)
+      return false if @os_type.nil?
```

Comment:
I added validation that type isn't `nil`, which was implied with the `EnumAttributeValidator` I removed.

Reply:
Same as above.